### PR TITLE
feat(video): added margin for the report button

### PR DESCRIPTION
**Pull request by austingardner**
## Description
adds a report-button class to the report button in ebay-video.

## Context
The view item team decided they wanted to have the report text in the control bar. 

## Screenshots
Screen Shot 2021-07-20 at 3 37 06 PM

### CODE REVIEW

File: dist/video/ds4/video.css [hunk 1]
```diff
@@ -56,3 +56,6 @@
 .video-player--poster .shaka-scrim-container {
   visibility: hidden;
 }
+.shaka-controls-button-panel .report-button > svg {
```
Comment (ianmcburnie) [inline dist/video/ds4/video.css:59]: Is `report-button` their class or ours? If ours it should conform to BEM, e.g. (`video-player__report-btn`)

Comment (ianmcburnie) [reply dist/video/ds4/video.css:59]: May as well also add support for RTL.

Comment (austingardner) [reply dist/video/ds4/video.css:59]: its ours, but it's not a direct descendant of video-player at all... would it be better to have something like 
`.shaka-controls-button-panel__report-btn`? Although shaka-controls-button-panel isn't ours

Comment (ianmcburnie) [reply dist/video/ds4/video.css:59]: Can you paste the full markup in the PR so I can get a better understanding.

Comment (austingardner) [reply dist/video/ds4/video.css:59]: Screen Shot 2021-07-20 at 4 05 10 PM

Comment (ianmcburnie) [reply dist/video/ds4/video.css]: It's a descendant of our `video-player` base class, so the double-underscore BEM syntax is appropriate. i.e. `video-player__report-button`

**Review by ianmcburnie — APPROVED** [review]

**Pull request merged**